setup.py

```diff
@@ -28,9 +28,54 @@
 from Cython.Build import cythonize
 from Cython.Distutils import build_ext
 import glob
-
 import numpy as np
 
+try:
+    from ctypes.utils import find_library
+except ImportError:
+    from ctypes.util import find_library
+
+
+def get_lib_suffix():
+    def walk_ld_library_path():
+        ld_library_path = os.environ.get('LD_LIBRARY_PATH', None)
```

Comment:
I think it not work for windows. If the DAAL is at the conda, then it will not be in this variable. Right?

Reply:
`lib_dir` search is added to cover this